.github/workflows/build_msi_installer.yml

```diff
@@ -108,6 +108,19 @@ jobs:
           setupType: promptflow_with_extra
           scriptPath: ${{ env.testWorkingDirectory }}
 
+      - name: Setup and Install dev promptflow
+        shell: pwsh
```

Comment:
Please trigger a release pipeline with the latest code and paste the link in the description.